akka-docs/src/test/java/jdocs/stream/operators/SourceOrFlow.java

```diff
@@ -475,6 +475,17 @@ void watchExample() {
     // #watch
   }
 
+  static CompletionStage<Integer> completionTimeoutExample() {
+    // #completionTimeout
+    Source<Integer, NotUsed> source = Source.range(1, 100000);
+    Flow<Integer, Integer, NotUsed> flow =
+        Flow.of(Integer.class).map(x -> x * x).completionTimeout(Duration.ofMillis(10));
+    CompletionStage<Integer> result =
+        source.via(flow).runWith(Sink.reduce((acc, element) -> acc + element), system);
+    return result;
```

Comment:
I wonder if we should avoid having `reduce` in here. Imagine if you are a newcomer who does not know what reduce does, then you'd have to visit the reduce docs before you could completely grasp this example. Perhaps just `source.map.completionTimeout.run` would be easier to understand?

Reply:
Sure, I'll change that